.gitignore

```diff
@@ -19,3 +19,6 @@ packages/_generated/src
 !.yarn/sdks
 !.yarn/versions
 coverage
+prometheus.yml
+monitoring/*
+docker-compose.yml
```

Comment:
I assume these are going to get added back in #195 so I'd just [hide them locally](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally#1753078)